app/models/account_login_stat.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+# It's important that the datetime columns on this table fit a "#{name}_at" pattern
+# where the name matches one of the service_names in the SAML::User::AUTHN_CONTEXTS hash.
+class AccountLoginStat < ApplicationRecord
+  # ['idme', 'myhealthevet', 'dslogon']
+  LOGIN_TYPES = SAML::User::AUTHN_CONTEXTS.map { |_k, v| v[:sign_in][:service_name] }.uniq.freeze
```

Comment:
This makes me think we could pull out constants in SAML::User, maybe into a file like `authn_constants.rb` or something like that, at least try to attempt to simplify/centralize these Authn definitions.

I wouldn't say that's a part of this PR, but maybe an idea for clarity in the future